packages/controls/src/widget_box.ts

```diff
@@ -91,18 +93,15 @@ export class BoxView extends DOMWidgetView {
     this.set_box_style();
   }
 
-  update_children(): void {
-    this.children_views
-      ?.update(this.model.get('children'))
-      .then((views: DOMWidgetView[]) => {
-        // Notify all children that their sizes may have changed.
-        views.forEach((view) => {
-          MessageLoop.postMessage(
-            view.luminoWidget,
-            Widget.ResizeMessage.UnknownSize
-          );
-        });
-      });
+  async update_children(): Promise<void> {
```

Comment:
Same comment, e.g. in line 92, we're calling this method but not awaiting on that, hence its possible the style is being set before the children are updated.
Logically that doesn't sem correct, though thats how it worked in the past.
Looking at the code now it seems incorrect to not await on this method when its called in other places.